package.json

```diff
@@ -30,23 +31,39 @@
   },
   "homepage": "https://github.com/Surnet/swagger-jsdoc",
   "dependencies": {
-    "chokidar": "^2.0.3",
-    "commander": "^2.11.0",
-    "doctrine": "^2.0.0",
-    "glob": "^7.1.2",
-    "js-yaml": "^3.8.4",
-    "recursive-iterator": "^2.0.3",
-    "swagger-parser": "^3.4.1"
+    "commander": "2.11.0",
+    "doctrine": "2.0.0",
+    "glob": "7.1.2",
+    "js-yaml": "3.8.4",
+    "recursive-iterator": "3.3.0",
+    "swagger-parser": "5.0.0"
   },
   "devDependencies": {
-    "body-parser": "^1.17.2",
-    "chai": "^4.0.2",
-    "express": "^4.15.3",
-    "istanbul": "^0.4.5",
-    "jscs": "^3.0.7",
-    "mocha": "^3.4.2",
-    "mocha-jscs": "^5.0.1",
-    "mocha-jshint": "^2.3.1",
-    "supertest": "^3.0.0"
+    "body-parser": "1.17.2",
+    "chai": "4.0.2",
+    "eslint": "5.1.0",
+    "eslint-config-airbnb-base": "13.0.0",
+    "eslint-config-prettier": "2.9.0",
+    "eslint-loader": "2.0.0",
+    "eslint-plugin-import": "2.13.0",
+    "eslint-plugin-prettier": "2.6.2",
+    "express": "4.15.3",
+    "husky": "1.0.0-rc.13",
+    "lint-staged": "7.2.0",
+    "mocha": "5.2.0",
+    "npm-run-all": "4.1.3",
+    "prettier": "1.13.7",
+    "supertest": "3.0.0"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
```

Comment:
isn't it a little bit confusing that linting also does the autoformatting?

Reply:
This is a very commonly see pattern nowadays. In fact, it's the first option suggested by prettier: https://prettier.io/docs/en/precommit.html#option-1-lint-staged-https-githubcom-okonet-lint-staged

Many other popular projects (such as CRA https://github.com/facebook/create-react-app/blob/next/package.json#L34) use this pattern in order to take off the need that you do formatting for rules which are aimed to avoid the needless discussions about formatting.

The main goal of adding prettier is to make the codebase more accessible while removing any unnecessary style discussions https://prettier.io/docs/en/why-prettier.html on top of the common eslint rules which are not only about formatting, but also catching issues and code smells with the language features.